docs/build-acceleration.md

```diff
@@ -61,6 +63,73 @@ Setting _Logging Level_ to a value other than `None` results in messages prefixe
 - `Minimal` produces a single message per out-of-date project.
 - `Info` and `Verbose` provide increasingly detailed information about the inner workings of the check, which are useful for debugging.
 
+### Validate builds are accelerated
+
+If build acceleration cannot be enabled for any of the reasons given below, builds continue to work as before but will be slower.
```

Comment:
I think you can just drop the "will be slower" part:

```suggestion
If build acceleration cannot be enabled for any of the reasons given below, builds continue to work as before.
```

They will be slower than if build acceleration could be enabled, but they won't be slower than they were before the build acceleration feature came along.